src/clients/mocks/MockSpokePoolClient.ts

```diff
@@ -67,10 +66,6 @@ export class MockSpokePoolClient extends SpokePoolClient {
     this.latestBlockSearched = blockNumber;
   }
 
-  addEvent(event: Event): void {
```

Comment:
Is this manual "push" fallback function useful at all/worth keeping? Or is it the programmer's "bug" if they're having to call this manual function

Reply:
It's been relocated to the `eventManager´ object, and that's public in this class so it is still available to use if needed - albeit a little bit less direct to get to.